app/jobs/appointment_reminders_job.rb

```diff
@@ -0,0 +1,28 @@
+class AppointmentRemindersJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Appointment.within_48_hours.each do |appointment|
+      send_reminder(appointment)
+    end
+  end
+
+  private
+
+  def send_reminder(appointment)
+    expected_reminder_count = appointment.audits.inject(0) do |count, audit|
```

Comment:
I wonder if this can be simplified just by asking the appointment if it has a reminder activity created within the same 48hr period? If it does then move along, otherwise we're good to send.